test/e2e/flask/multichain-api/evm/wallet_createSession.spec.ts

```diff
@@ -443,6 +457,9 @@ describe('Multichain API', function () {
             ),
           );
 
+          // Issue #38699: Account 2 is connected but it's not imported in my wallet
+          await testDapp.checkConnectedAccounts([ACCOUNT_1]);
```

Comment:
ℹ️  I found this bug while adding this new assert, as a second account appeared. Opened a ticket here:

https://github.com/MetaMask/metamask-extension/issues/38699